lib/src/widgets/load_earlier.dart

```diff
@@ -2,24 +2,20 @@ part of dash_chat;
 
 class LoadEarlierWidget extends StatelessWidget {
   const LoadEarlierWidget({
-    Key key,
-    @required this.onLoadEarlier,
-    @required this.defaultLoadCallback,
+    Key? key,
+    required this.onLoadEarlier,
+    required this.defaultLoadCallback,
   }) : super(key: key);
 
-  final Function onLoadEarlier;
+  final Function? onLoadEarlier;
```

Comment:
If it's required then the variable can't be non-nullable

Reply:
nice catch!